Create private-browser.md (#626)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/features/protection/private-browser.md

---
title: Private browser
sidebar_position: 7
---

:::info

This article is about AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, [download the AdGuard app](https://agrd.io/download-kb-adblock)

:::

In v4.7, AdGuard for Android introduced an in-app private browser. You can access it from the main screen by tapping the *Private browser* button or from the *Protection* tab, where you can find some customization options:

![Private browser](https://cdn.adtidy.org/content/release_notes/ad_blocker/android/v4.7/agpb_en.png)

> 

Reviewer: Maybe before listing the functions for better readability you should write for example "Here's what you can configure:"?

> - To set a default search engine, click *Default search engine* and choose between the available options: DuckDuckGo (default), Google, Bing, Yahoo, Baidu, or Yandex

Reviewer: It would be better to replace click with tap if we have already mentioned the word "tapping" earlier

> - To create a browser shortcut, click *Create browser widget* and follow the on-screen instructions

Reviewer: It would be better to replace click with tap if we have already mentioned the word "tapping" earlier

---
title: Private browser
sidebar_position: 7
---

:::info

This article is about AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, [download the AdGuard app](https://agrd.io/download-kb-adblock)

:::

> Starting with v4.7, AdGuard for Android introduced an in-app private browser. You can access this feature from the main screen by tapping the *Private browser* button.

Reviewer 2: ```suggestion
In v4.7, AdGuard for Android introduced an in-app private browser. You can access it from the main screen by tapping the *Private browser* button or from the *Protection* tab, where you can find some customization options:.
```

![Private browser](https://cdn.adtidy.org/content/release_notes/ad_blocker/android/v4.7/agpb_en.png)

> The browser is also accessible through the *Protection* tab, where you can find some customization options:

Reviewer 2: ```suggestion
```

> - To set a default search engine, click *Default search engine* and choose between the available options: DuckDuckGo (default), Google, Bing, Yahoo, Baidu or Yandex

Reviewer 2: ```suggestion
- To set a default search engine, click *Default search engine* and choose between the available options: DuckDuckGo (default), Google, Bing, Yahoo, Baidu, or Yandex
```

> - To create a shortcut to the browser, click *Create browser widget* and follow the instructions on the screen

Reviewer 2: ```suggestion
- To create a browser shortcut, click *Create browser widget* and follow the on-screen instructions
```

> - To get suggestions from your search engine instead of typing the entire query, toggle the option *Search suggestions*

Reviewer 2: ```suggestion
- To get suggestions from your search engine instead of typing the entire query, toggle the *Search suggestions* option
```

:::note

> The private browser is still in the early stages of development and has a few limitations. For now, we recommend using it as a supplement to your usual browser’s Incognito Mode, rather than a replacement.

Reviewer 2: ```suggestion
The private browser is still in the early stages of development and has some limitations. For now, we recommend using it as a supplement to your browser’s Incognito mode, rather than as a replacement.
```

:::

> In v4.7, AdGuard for Android introduced an in-app private browser. You can access it from the main screen by tapping the *Private browser* button or from the *Protection* tab, where you can find some customization options:.

Reviewer 2: ой, здесь точка лишняя в конце
